Project/MyGameLibrary/MyGameLibrary.csproj

```diff
@@ -9,9 +9,10 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>MyGameLibrary</RootNamespace>
     <AssemblyName>MyGameLibrary</AssemblyName>
-    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
```

Comment:
Why has the framework version been downgraded?

Reply:
My mistake, this is from using an older version of VS on my machine. It shouldn't change any functionality, but I'll update this before I push my updated code.